standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1397,9 +1397,6 @@ public static ConfVars getMetaConf(String name) {
             " If org.apache.hive.hcatalog.listener.DbNotificationListener is configured along with other transactional event" +
             " listener implementation classes, make sure org.apache.hive.hcatalog.listener.DbNotificationListener is placed at" +
             " the end of the list."),
-    TRUNCATE_ACID_USE_BASE("metastore.acid.truncate.usebase", "hive.metastore.acid.truncate.usebase", false,
```

Comment:
How old is this config? Is this released somewhere? Shall we deprecate it, or it is ok to remove it?

Reply:
it's disabled downstream + it doesn't work correctly as the property activating it is not propagated.